articles/architecture-scenarios/b2c/user-profile-mgmt.md

```diff
@@ -0,0 +1,132 @@
+---
+title: User Profile Management
+description: Learn about user profile management planning considerations for your B2C implementation. 
+toc: true
+topics:
+    - b2c
+    - ciam
+    - user-profiles
+contentType: concept
+useCase:
+  - profile-management
+---
+# User Profile Management
+
+User information is stored in a [*user profile*](/users/concepts/overview-user-profile) and can come from a variety of sources such as [identity providers](/identityproviders), your own databases, and enterprise connections (Active Directory, SAML, etc.). You can *normalize* user data that comes from a variety of data sources. By default, Auth0 creates one user profile for each user identity.
+
+Here are some examples of the things that determine how you will manage user profile information:
```

Comment:
In some other sections - User Logout for example (articles/architecture-scenarios/b2c/user-logout.md) - this seems to have been placed under a `### Design considerations` section. Again, there seems to be a lack of consistency with regards to the formatting here.